docs/development.md

```diff
@@ -0,0 +1,222 @@
+# Development
```

Comment:
`Dev server`, `Staging server`, `Client scripts` are all explained in their respective sections.